dataset/dictionary/enemies.json5

```diff
@@ -959,52 +1119,69 @@
   {
     en: "Rifthound Whelp",
     ja: "獣域ウェルプ",
+    zh: "兽境幼兽",
     pronunciationJa: "じゅういきウェルプ",
+    variants: {
+      zh: [ "小流血狗" ],
+    },
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound Whelp",
     ja: "獣域ウェルプ・岩",
+    zh: "兽境幼兽·岩",
     pronunciationJa: "じゅういきウェルプ・いわ",
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Thundercraven Rifthound Whelp",
     ja: "獣域ウェルプ・雷",
+    zh: "兽境幼兽·雷",
```

Comment:
This seems wrong translation.

![](https://user-images.githubusercontent.com/107226297/174476629-0f665088-79e6-48e0-8369-6cec0d7bee3f.png)